### DISCUSSION TITLE
adding responsiveness

### DISCUSSION THREAD

**Discussion by quebesinmars** [aristocratos/btop#527] [Ideas]
it would be such an amazing feature :D

**Comment by deckstose** [comment]
Can you elaborate on that?

**Reply by quebesinmars** [reply]
Hi , i think since BTOP++ can know the dimensions of my terminal window then it's possible to change the layout instead of showing the message :  "need the current config Width:80 Height = 24 "

**Reply by deckstose** [reply]
Yeah but it might not be reasonable to draw something to a 10x10 terminal because you can't read from it.
And even if you might disable a box, in what order should that be? How is that determined. That are some difficult questions we need answers for before implementing that.

It's probably best to leave this message as is.
